x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/searchablesnapshots/SearchableSnapshotsConstants.java

```diff
@@ -38,23 +32,7 @@ public static boolean isSearchableSnapshotStore(Settings indexSettings) {
     public static boolean isPartialSearchableSnapshotIndex(Map<Setting<?>, Object> indexSettings) {
         assert indexSettings.containsKey(INDEX_STORE_TYPE_SETTING) : "must include store type in map";
         assert indexSettings.get(SNAPSHOT_PARTIAL_SETTING) != null : "partial setting must be non-null in map (has default value)";
-        return SNAPSHOT_DIRECTORY_FACTORY_KEY.equals(indexSettings.get(INDEX_STORE_TYPE_SETTING))
+        return SearchableSnapshotsSettings.SEARCHABLE_SNAPSHOT_STORE_TYPE.equals(indexSettings.get(INDEX_STORE_TYPE_SETTING))
             && (boolean) indexSettings.get(SNAPSHOT_PARTIAL_SETTING);
     }
-
-    public static boolean isPartialSearchableSnapshotIndex(Settings indexSettings) {
-        return SNAPSHOT_DIRECTORY_FACTORY_KEY.equals(INDEX_STORE_TYPE_SETTING.get(indexSettings))
-            && SNAPSHOT_PARTIAL_SETTING.get(indexSettings);
-    }
-
-    public static final String CACHE_FETCH_ASYNC_THREAD_POOL_NAME = "searchable_snapshots_cache_fetch_async";
-    public static final String CACHE_FETCH_ASYNC_THREAD_POOL_SETTING = "xpack.searchable_snapshots.cache_fetch_async_thread_pool";
-
-    public static final String CACHE_PREWARMING_THREAD_POOL_NAME = "searchable_snapshots_cache_prewarming";
-    public static final String CACHE_PREWARMING_THREAD_POOL_SETTING = "xpack.searchable_snapshots.cache_prewarming_thread_pool";
-
-    public static final String SNAPSHOT_BLOB_CACHE_INDEX = ".snapshot-blob-cache";
-
-    public static final Version SHARED_CACHE_VERSION = Version.V_7_12_0;
```

Comment:
This has been removed from x-pack core and placed into the searchable snapshot plugin instead.